index.bs

```diff
@@ -151,9 +151,8 @@ Note that we're explicitly monkeypatching CSSOM. We are working on [merging this
     <dd>
     	1. Let |sheet| be this {{/CSSStyleSheet}} object.
 		2. If |sheet|'s [=constructed flag=] is not set, or |sheet|'s [=disallow modification flag=] is set, throw a "{{NotAllowedError}}" {{DOMException}}.
-		3. Set |sheet|'s [=CSS rules=] to an empty list, and set |sheet|'s [=disallow modification flag=].
-		4. Let |promise| be a promise.
-		5. [=In parallel=], do these steps:
+		3. Let |promise| be a promise.
```

Comment:
The above is also wrong:
> If |sheet|'s [=constructed flag=] is not set, or |sheet|'s [=disallow modification flag=] is set, throw a "{{NotAllowedError}}" {{DOMException}}.

Should be:

> If |sheet|'s [=constructed flag=] is not set, or |sheet|'s [=disallow modification flag=] is set, return a promise rejected with "{{NotAllowedError}}" {{DOMException}}.

Reply:
Why? That can totally throw instead of returning a promise.

Reply:
On my phone, so quick response: Things the return promises shouldn’t throw, they should only return promises (which are rejected). See TAG’s API guidelines or probably also something in WebIDL spec. One of the two talks about this.

Reply:
Example 52 in the WebIDL spec shows this.

Reply:
Fair enough, though I guess that we need to check whether Blink follows the spec in this case (looks like it does not, which means it agrees with your suggestion... yay?).

Reply:
For reference: https://www.w3.org/2001/tag/doc/promises-guide#always-return-promises

Reply:
I'm going to make a new PR that will change this to always return a promise. @marcoscaceres Thanks for bringing this to light.

Reply:
Closing this PR in favor of new PR for following the TAG guidelines.